### ISSUE TITLE
Syntax error with reserved word as column name

### ISSUE THREAD

**Issue by michaeldg** [maxbube/mydumper#240]
Hi,

Can you please add escaping for column names? Mydumper fails when there is a table with a column called `primary`:
** (mydumper:12990): CRITICAL **: 08:54:11.620: Error dumping table (table_prd.users) data: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'primary,active,is_in_centre,has_login,has_active_login,is_my_site_admin,origin,' at line 1 

Thank you,
Michaël

**Comment by fredricj** [comment]
It would be helpful with the CREATE TABLE statement

**Comment by michaeldg** [comment]
Ofcourse, apologies!
```
CREATE TABLE `users` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `external_reference_id` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `email` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
  `password` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `remember_token` varchar(100) COLLATE utf8_unicode_ci DEFAULT NULL,
  `first_name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `last_name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `full_name` varchar(255) GENERATED ALWAYS AS (concat_ws(' ',`first_name`,`last_name`)) STORED,
  `phone` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `address` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `primary` tinyint(1) NOT NULL DEFAULT 0,
  `active` tinyint(1) NOT NULL DEFAULT 1,
  `is_in_centre` tinyint(1) NOT NULL DEFAULT 0,
  `has_login` tinyint(1) NOT NULL DEFAULT 0,
  `has_active_login` tinyint(1) NOT NULL DEFAULT 0,
  `is_my_site_admin` tinyint(1) NOT NULL DEFAULT 0,
  `origin` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `created_at` timestamp NULL DEFAULT NULL,
  `updated_at` timestamp NULL DEFAULT NULL,
  `deleted_at` timestamp NULL DEFAULT NULL,
  `verified` tinyint(1) NOT NULL DEFAULT 0,
  `token` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `some_id` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `blocked` tinyint(1) NOT NULL DEFAULT 0,
  `is_staff` tinyint(1) NOT NULL DEFAULT 0,
  `last_login_date` datetime DEFAULT NULL,
  `date_format` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `type` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `some_contact_id` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `users_type_external_reference_id_unique` (`type`,`external_reference_id`),
  KEY `users_email_index` (`email`),
  KEY `users_some_id_index` (`some_id`),
  KEY `users_is_staff_index` (`is_staff`),
  KEY `users_primary_index` (`primary`),
  KEY `users_active_index` (`active`),
  KEY `users_is_in_centre_index` (`is_in_centre`),
  KEY `users_has_login_index` (`has_login`),
  KEY `users_has_active_login_index` (`has_active_login`),
  KEY `users_is_my_regus_admin_index` (`is_my_regus_admin`),
  KEY `users_external_reference_id_index` (`external_reference_id`),
  KEY `users_external_reference_id_type_index` (`external_reference_id`,`type`),
  KEY `users_full_name_index` (`full_name`),
  KEY `users_some_contact_id_index` (`some_contact_id`)
) ENGINE=InnoDB AUTO_INCREMENT=1316930 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci
```

**Comment by fredricj** [comment]
Same issue as #215

**Comment by maxbube** [comment]
Already fixed by #215
